src/components/modules/EmailCode/EmailCode.view.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <form @submit.prevent="onSubmit">
+    <v-title>
+      {{ $t('components.emailCode.title') }}
+    </v-title>
+    <v-description>
+      <span v-html="$t('components.emailCode.description', { email })" />
+    </v-description>
+    <form-item>
+      <v-input
+        v-model="code"
+        v-validate="'required|digits:6'"
+        data-vv-as="code"
+        data-vv-name="code"
+        :error="errors.first('code') || error"
+        name="code"
+        :placeholder="$t('components.emailCode.placeholder')"
+        data-test="code-input"
+      />
+    </form-item>
+    <form-item class="v-mb-24">
+      <v-button
+        :disabled="!isFormValid || isLoading"
+        :is-loading="isLoading"
+        type="submit"
+        data-test="submit-button"
+      >
+        {{ $t('global.confirm') }}
+      </v-button>
+    </form-item>
+    <form-row class="v-fs-14 v-text-center">
+      <send-code
+        :is-loading="isLoading"
+        @click="sendCode"
+      />
+    </form-row>
+  </form>
+</template>
+
+<script>
+import VButton from '@endpass/ui/kit/VButton';
+import VInput from '@endpass/ui/kit/VInput';
+import FormItem from '@/components/common/FormItem';
+import FormRow from '@/components/common/FormRow';
+import formMixin from '@/mixins/form';
+import VTitle from '@/components/common/VTitle';
+import VDescription from '@/components/common/VDescription';
+import SendCode from '@/components/common/SendCode';
+
+export default {
+  name: 'EmailCodeView',
+
+  props: {
+    email: {
+      type: String,
+      required: true,
+    },
+
+    isLoading: {
+      type: Boolean,
+      default: false,
+    },
+
+    error: {
+      type: String,
+      default: '',
+    },
+  },
+
+  data: () => ({
+    code: '',
+  }),
+
+  watch: {
+    error(msg) {
+      if (!msg) {
+        this.$validator.errors.removeById('sendCodeId');
+        return;
+      }
+
+      this.$validator.errors.add({
```

Comment:
So, what about many errors and fields?

Reply:
I didn't see any cases. We use error for submit